hbase-hadoop-compat/src/test/java/org/apache/hadoop/hbase/regionserver/MetricsTableWrapperStub.java

```diff
@@ -117,13 +117,19 @@ public long getCpRequestsCount(String table) {
   public Map<String, Long> getMemstoreOnlyRowReadsCount(String table) {
     Map<String, Long> map = new HashMap<String, Long>();
     map.put("table#info", 3L);
+    map.put("table##", 3L);
```

Comment:
Do we have some assert where it make sure the metric is there with key <table name>#<cf name>?

Reply:
This key is of internal mapping, we dont have any exposed metric where we separate table & CF using #. So I think no asset in that case (please correct me if I'm wrong).